api/document/js/__tests__/menu.test.ts

```diff
@@ -0,0 +1,34 @@
+import { flatten } from 'lodash/array';
+import { EditorView } from 'prosemirror-view';
+import schema, { factory } from '../schema';
+import { EditorState } from 'prosemirror-state';
+import menu from '../menu';
+import { Api, JsonApi } from '../Api';
+
+describe('menu functions', () => {
+  it('has the expected buttons', () => {
```

Comment:
Should this be "_have_ the expected buttons"?  I'm just trying to read out the first args of `describe()` and `it()` in a sentence, and it comes out to "menu functions has the expected buttons" which sounds a bit funky.

Reply:
Maybe s/menu functions/menu function? `it('have ` also isn't quite right...

Reply:
This is an amusing set of comments to find… I was reading the “sentences” purely as “it…” and not as including the argument to `describe`. I think the easiest change to make here is to `s/menu functions/menu`.